Client/Frontend/Sync/BraveCore/History/Historyv2.swift

```diff
@@ -0,0 +1,215 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import Data
+import BraveRewards
+import BraveShared
+import CoreData
+import Shared
+
+private let log = Logger.browserLogger
+
+// A Lightweight wrapper around BraveCore history
+// with the same layout/interface as `History (from CoreData)`
+class Historyv2: WebsitePresentable {
+    
+    /// Sections in History List to be displayed
+    enum Section: Int, CaseIterable {
+        /// History happened Today
+        case today
+        /// History happened Yesterday
+        case yesterday
+        /// History happened between yesterday and end of this week
+        case lastWeek
+        /// History happaned
```

Comment:
typo, and it happened when exactly? can update this comment

Reply:
Fix at https://github.com/brave/brave-ios/pull/3703/commits/859375beafc1a0b1778a58dd79cb21c16bacd6cf